components/next_steps.js

```diff
@@ -81,6 +81,11 @@ export class NextSteps extends Component {
 
     return (
       <div className={outerDiv}>
+        <a
```

Comment:
did you try adding the "next-steps" id to the header for the section? if so did it work? I think that would be preferable to a hidden element.

Reply:
It does work, I'll make that chance before merging.